README.md

```diff
@@ -109,11 +78,70 @@ s3://mybucket/aws-instance-scheduler/v1.4.0/instance-scheduler.template
 s3://mybucket/aws-instance-scheduler/v1.4.0/instance-scheduler-remote.template
 ```
 
-## Deploy
+### Deploy
 
 See the [AWS Instance Scheduler Implementation Guide](https://s3.amazonaws.com/solutions-reference/aws-instance-scheduler/latest/instance-scheduler.pdf) for deployment instructions, using the link to the instance-scheduler.template from your bucket, rather than the one for AWS Solutions. Ex. https://mybucket.s3.amazonaws.com/aws-instance-scheduler/v1.4.0.mybuild/instance-scheduler.template
 
-## CDK Documentation
+
+# Testing the Solution
+
+## Running Tests Locally
+to test the solution you will need to install tox
+
+```pip install tox```
+
+Then from the root directory of the solution
+```
+//run all unit tests
+tox
+
+//test just the lambda code
+tox -e lambda
+
+//test just the cdk code
+tox -e cdk
+```
+
+## Automated Testing Pipeline
+AWS Instance Scheduler comes with an automated testing pipeline that can be used to automatically test any changes you
+develop for the solution on your own development fork
+
+#### _Prerequisites - You must have an AWS account and a fork of this repo_
```

Comment:
I would put the content of this line within the section.

Reply:
I don't think I follow. Though I have flipped the order so that this comes before the explanation which does seem to flow better.

Reply:
This is a fourth level heading, presumably beginning a section about what the prerequisites are for the automated testing pipeline. But the content is within the line itself. This is the only place in the file where that's the case. Why not put the information from that line into the section that the heading begins?
```md
#### Prerequisites
- an AWS account
- a fork of this repo
```